app.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const path = require('path');
+
+const bodyParser = require('body-parser');
+
+const index = require('./routes/index');
+const users = require('./routes/users');
+
+const app = express();
+
+// view engine setup
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'jade');
```

Comment:
@meghanprestemon please confirm that your project will be using `jade` or remove this line, thanks!